cai_causal_graph/type_definitions.py

```diff
@@ -69,6 +69,10 @@ def __str__(self) -> str:
 PAIR_T = Tuple[str, str]  # Defines a pair of variables. Can be used to query edges, paths, functions.
 NodeLike = Union[str, HasIdentifier]
 
+# meta for time series
+TIME_LAG = 'time_lag'
+VARIABLE_NAME = 'variable_name'
```

Comment:
there are a few cases where this is not used. could you please do a search and use this constants instead?